scripts/package.ts

```diff
@@ -127,6 +131,23 @@ class Builder {
       break;
     }
 
+    // If we have files (resp. extraFiles / extraResources) both at the top
```

Comment:
Would it be clearer to change this to
```
// When there are files (e.g., extraFiles or extraResources) specified at both 
// the top-level and platform-specific levels, we need to combine them 
// and place the combined list at the top level. This approach enables us to have 
// platform-specific exclusions, since the two lists are initially processed 
// separately and then merged together afterward.
```